packages/relay/src/lib/eth.ts

```diff
@@ -481,7 +479,7 @@ export class EthImpl implements Eth {
     const requestIdPrefix = formatRequestIdMessage(requestId);
     this.logger.trace(`${requestIdPrefix} gasPrice()`);
     try {
-      let gasPrice: number | undefined = this.cache.get(constants.CACHE_KEY.GAS_PRICE);
+      let gasPrice: any = this.cache.get(constants.CACHE_KEY.GAS_PRICE);
 
```

Comment:
Is type safety not a concern here anymore?  getFreeWeibars still returns a number, or am I missing something?

Reply:
There were a lot of changes, and finally, it's ready for review again :upside_down_face:.